ledger/sandbox-on-x/reference.conf

```diff
@@ -36,6 +47,8 @@ ledger {
 
     # Defines the type of registry being created for each Participant configuration.
     # Options: jvm-shared, new
+    # TODO consolidate with Canton report-jvm-metrics=yes: https://github.com/DACH-NY/canton/blob/main/community/app/src/main/scala/com/digitalasset/canton/metrics/MetricsFactory.scala#L24
```

Comment:
this is something else, this parameter says how many different parameter registries are created in case multiple components are mushed into a single jvm